### ISSUE TITLE
[MODERATE] [region] All page content should be contained by landmarks

### ISSUE THREAD

**Issue by petermakowski** [canonical-web-and-design/maas-ui#3645]
Multiple pages violate the landmarks rule. 
https://dequeuniversity.com/rules/axe/4.3/region?application=axeAPI

Some of the elements that cause the violation:
- `<div class="section__header-wrapper">`
- `<div class="p-status-bar"><div class="row"><div class="col-6"><strong data-testid="status-bar-maas-name">ubuntu-linux-20-04-desktop MAAS</strong>: <span data-testid="status-bar-version">3.1.0</span></div></div></div>`
